Lib/importlib/_bootstrap.py

```diff
@@ -361,6 +361,7 @@ def __init__(self, name, loader, *, origin=None, loader_state=None,
         self.origin = origin
         self.loader_state = loader_state
         self.submodule_search_locations = [] if is_package else None
+        self.uninitialized_submodules = []
```

Comment:
This should better be a set, no? That way you can use `PySet_Contains` in the C code

Reply:
Sure, I don't really have any preference, as this is something that will not really matter much. Let me know if you want me to change it to set.